src/material-experimental/mdc-tabs/tab-group.scss

```diff
@@ -0,0 +1,47 @@
+@import '@material/tab/mixins';
+@import '@material/ripple/variables';
+@import '../../material/core/style/variables';
+@import '../../material/core/style/noop-animation';
+@import '../mdc-helpers/mdc-helpers';
+@import './tabs-common';
+
+@include mdc-tab-without-ripple($query: $mat-base-styles-query);
+@include tab-ripple;
+
+.mat-mdc-tab {
+  @include tab;
+
+  // Note that we only want to target direct descendant tabs. Also note that
+  // `mat-stretch-tabs` is part of the public API so it should not be changed to `mat-mdc-`.
+  .mat-mdc-tab-group[mat-stretch-tabs] > .mat-mdc-tab-header & {
+    flex-grow: 1;
+  }
+}
+
+// MDC doesn't support disabled tabs so we need to improvise.
```

Comment:
We should file this as a feature request (or bug) since `aria-disabled` is a supported state for `role="tab"`

Reply:
My guess is that it's not there, because it's not in the spec either: https://material.io/components/tabs/#states.